### PR TITLE
Updated "Add Friends" Button

### PR DESCRIPTION
**Pull request by malena-3**
**Step One:**
1) From the profiles table, click the insert button.
2) Click the insert column

Screenshot 2024-07-31 at 2 01 52 PM

**Step Two:**
3) Name it “friend_ids”
4) Type will be T uuid
5) Check of Define as Array

Screenshot 2024-07-31 at 2 14 22 PM

**Step Three:**
6) Constraints - Allow for it to be nullable
7) Click save to complete !

Screenshot 2024-07-31 at 2 15 51 PM

### CODE REVIEW

**Comment by malena-3** [comment]
Step One:
Screenshot 2024-07-31 at 2 01 52 PM

1) From the profiles table, click the insert button.
2) Click the insert column

Step Two:
Screenshot 2024-07-31 at 2 14 22 PM

3) Name it “friend_ids”
4) Type will be T uuid
5) Check of Define as Array

Step Three:
Screenshot 2024-07-31 at 2 15 51 PM

6) Constraints - Allow for it to be nullable
7) Click save to complete !

File: src/screens/AddFriendScreen.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+              'Julie',
+            ],
+          },
+        ]}
+        renderItem={({item}) => <Text style={styles.item}>{item}</Text>}
+        renderSectionHeader={({section}) => (
+          <Text style={styles.sectionHeader}>{section.title}</Text>
+        )}
+        keyExtractor={item => `basicListEntry-${item}`}
+      /> */}
```
Comment (bakerfugu) [inline src/screens/AddFriendScreen.js]: Add a comment explaining what this is / why this commented code was left here

File: src/screens/ChatScreen.js [hunk 2]
```diff
@@ -64,7 +64,7 @@ export default function ChatScreen({ navigation }) {
         },
       ]}
     >
-      <Header title="Chat" />
+      <Header title="Chattt" />
```
Comment (bakerfugu) [inline src/screens/ChatScreen.js]: change back to `Chat`

**Review by bakerfugu — CHANGES_REQUESTED** [review]
please review comments and commit with new changes

**Comment by bakerfugu** [comment]
And move instructions with images to top comment so it's easier for people to see

**Review by bakerfugu — APPROVED** [review]
Looks good

File: src/screens/AddFriendScreen.js [hunk 3]
```diff
@@ -45,43 +43,20 @@ const styles = StyleSheet.create({
...
+      <Text style={{ textAlign: "center", fontSize: 25, marginTop: 20 }}>
+        Quick Add
+      </Text>
+      <ScrollView
+        horizontal={false}
+        showsHorizontalScrollIndicator={false}
+        contentContainerStyle={styles.stories}
+      >
+        <AddFriendBitmoji />
+      </ScrollView>
```
Comment (malena-3) [inline src/screens/AddFriendScreen.js:56]: Commented out code is an additional section for the "Add Friends" page. A list of contacts with pfps and add button.

**Pull request merged**